sdk/storage/storage-internal-avro/src/IReadable.ts

```diff
@@ -0,0 +1,67 @@
+export abstract class IReadable {
```

Comment:
>IReadable [](start = 22, length = 9)

It's confusing to use "IReadable" which conflicts with "Readable" in stream. As this interface is only used by Avro parser, it's better to use like "IAvroReadable"

Reply:
Drop the I prefix too, it's not a TypeScript convention. AvroReadable seems good to me!

Reply:
Renamed IReadable to AvroReadable.
And separated the ReadableFromStream implementation with the interface declaration as we need a different implementation in browsers.